apisix/cli/ngx_tpl.lua

```diff
@@ -447,6 +447,7 @@ http {
 
     init_worker_by_lua_block {
         apisix.http_init_worker()
+        apisix.stream_init_worker()
```

Comment:
we don't need this

Reply:
I want to explain my code design ：
       This is using logic about stream_routes inside admin_api(**/apisix/admin**).    Get  stream_routes from   apisix.stream.router.ip_port.routes() and apisix.stream.router.ip_port.routes()  depends on stream_init_worker().  So under the  scope of admin_api , I have to add  stream_init_worker in advance。